addons/website/views/snippets/s_map.xml

```diff
@@ -53,7 +53,7 @@
             <we-checkbox string="Description" data-no-preview="true" data-show-description="true"/>
         </div>
     </xpath>
-    <xpath expr="//div[@data-option-name='minHeight']" position="inside">
+    <xpath expr="//div[@data-js='ScrollButton']//we-button-group" position="after">
```

Comment:
After this change, you add an option inside an option... if it does not crashes, it is pure luck

Reply:
Why are you adding a smiley which laughs at my comment and keep the PR unchanged..? What am I missing? :grimacing: 

It is actually already an error in the original code but fix it for master please :grimacing: